src/sentry/issues/endpoints/group_event_details.py

```diff
@@ -161,6 +161,10 @@ def get(self, request: Request, group: Group, event_id: str) -> Response:
         """
         environments = [e for e in get_environments(request, group.project.organization)]
         environment_names = [e.name for e in environments]
+        # The streamlined UI doesn't want a fallback if a query has no results.
+        should_always_return_event = not features.has(
+            "organizations:issue-details-streamline", group.organization, actor=request.user
+        )
```

Comment:
this would also enable it for the old ui if the org has the feature. do we need to do anything about that

Reply:
Refactored a whole bunch, the new querying/date-ranges are not behind the flag, though we can have the frontend only send those query params when we have the flag.